scrapy/linkextractors/lxmlhtml.py

```diff
@@ -20,8 +18,9 @@
 
 _collect_string_content = etree.XPath("string()")
 
+
 def _nons(tag):
-    if isinstance(tag, basestring):
+    if isinstance(tag, six.string_types):
```

Comment:
@kmike , no I can't.
This code was ~~copied~~ borrowed from `lxml`: https://github.com/lxml/lxml/blob/master/src/lxml/html/__init__.py#L126